src/Globalstyle.tsx

```diff
@@ -29,6 +29,12 @@ const GlobalStyle = createGlobalStyle`
   font-style: normal;
   }
 
+  @media all and (max-width: 750px) {
+  html {
+    font-size: 85%; 
+  }
+}
```

Comment:
```suggestion
  @media all and (max-width: 750px) {
  html {
    font-size: 85%; 
    }
  }
```
여기 간격이 이상하네 이렇게 수정해주세용

Reply:
제가 빠르게 해치웠습니다..!

Reply:
감사합니다~~